loginUi/src/main/kotlin/com/gravatar/app/loginUi/presentation/login/LoginViewModel.kt

```diff
@@ -41,10 +41,10 @@ internal class LoginViewModel(
         viewModelScope.launch {
             _uiState.update {
                 it.copy(
-                    error = null
+                    error = null,
+                    isLoading = false,
                 )
             }
-            sendAction(LoginAction.StartOAuth)
```

Comment:
⛏️  Not directly related to this PR, but notice this as we remove the action, and the test file is not modified. We can add some tests to cover `OnTryAnotherAccountClicked` and `OnLoginClicked` events.

Reply:
Done!